src/pages/[nickname]/otherProfilePage.tsx

```diff
@@ -10,10 +10,9 @@ export const OtherProfilePage = () => {
   const { data, isLoading } = useGetProfile({
     nickname: nicknameParams,
   });
-  const { data: myFollowingIdsMap, isLoading: isMyFollowingIdsMapLoading } =
-    useGetMyFollowingIdsMap();
+  const { data: myProfile, isLoading: isMyProfileLoading } = useGetMyProfile();
 
-  if (!data || isLoading || !myFollowingIdsMap || isMyFollowingIdsMapLoading) {
+  if (!data || isLoading || !myProfile || isMyProfileLoading) {
```

Comment:
isMyProfileLoading이랑 myProfile을 조건문에 넣은 이유가 있을까요?
두 개의 값이 동시에 바뀌기 때문에 하나만 넣어도 될 것 같은데, 둘 다 넣은 이유가 궁금합니다!

Reply:
빈 객체를 넣어주게 되면 밑에 있는 

```tsx
  const { followersIds, followingsIds, pet, userId } = data;
``` 

에서 타입이 좀 깨지더라고요 

기본 값으로 {follwerIds : [] , ...} 넣어주는 것 보단 저게 나아보여서 이렇게 했숩니다 좋은 방법이 있을까요

Reply:
myProfile이랑 data랑 연관이있나요?

myProfile 데이터가 불러오기 전까지 isMyProfileLoading가 true여서 하나만 넣어도 되지 않을까 해서요.

```suggestion
  if (!data || isLoading || !myProfile) {
```

Reply:
이런 제가 설명을 잘못했군요 

![image](https://github.com/user-attachments/assets/af688c40-5f1f-4c05-a967-44cd20481758)

다만 그러더라도 이 부분 때문에 타입 검증은 지울 수 없습니다.